### ISSUE TITLE
Why Observer is not binding target? 

### ISSUE THREAD

**Issue by AlexeyShvachka** [ReactiveCocoa/ReactiveSwift#576]
tl;dr: I want to match behaviour of `PublishSubject` from RxSwift using `Signal.pipe()`

I want to pass to pass events that my Views emits to ViewModel and `MutableProperty` doesn't seem like a right choice. 

I don't need to store current value, and `MutableProperty` should be initialised with some value, which I will not have at ViewModel creation. Action seems like overkill too, because there is no return value. 

I'm thinking of creating pipe and binding user input to it, but observer itself is not binding  #target.
Is there any problem with using pipe like that or is it just not yet implemented?

**Comment by andersio** [comment]
`<~` is supposed to weakly reference both ends, and be ton down when the lifetime of either ends has ended. `Observer` alone cannot satisfy these requirements, so it wasn’t added initially.

We may consider allowing bindings towards a pipe yielded by `Signal.pipe` though.
